src/Analyzers/CSharp/CodeFixes/UseImplicitOrExplicitType/UseExplicitTypeCodeFixProvider.cs

```diff
@@ -158,5 +210,18 @@ private static ExpressionSyntax GenerateTupleDeclaration(ITypeSymbol typeSymbol,
                 SyntaxFactory.Token(SyntaxKind.CloseParenToken))
                 .WithTrailingTrivia(parensDesignation.GetTrailingTrivia());
         }
+
+        private static SyntaxNode GenerateTypeDeclaration(TypeSyntax typeSyntax, ITypeSymbol newTypeSymbol)
+        {
+            // We're going to be passed through the simplifier.  Tell it to not just convert this back to var (as
+            // that would defeat the purpose of this refactoring entirely).
+            var newTypeSyntax = newTypeSymbol
+                         .GenerateTypeSyntax(allowVar: false)
+                         .WithTriviaFrom(typeSyntax);
+
+            Debug.Assert(!newTypeSyntax.ContainsDiagnostics, "Explicit type replacement likely introduced an error in code");
```

Comment:
this assert isn't really valuable.  the only real way it could happen is if the prior trivia had a diagnostic *which is possible*.  But in that case, it would be fine to copy that over.  i'm 100% ok deleting this line.